app/assets/stylesheets/themes/dark.css

```diff
@@ -1,3 +1,8 @@
+::selection {
+  background-color: rgba(248,28,229, 0.5); /* 0.5 represents 50% opacity */
```

Comment:
We might want to avoid hard coded colors. You can use a theme color like so.

```css
background-color: color-mix(in srgb, var(--btn-primary-bg), transparent 30%);
```